### PR TITLE
Adding webpack and babel

### PR DESCRIPTION
**Pull request by bandax**
- Configure webpack to run with dev and prod environment.
- Add the first jest test
- Configure scripts to run webpack on different environments
- Include support for typescript and scss

### CODE REVIEW

File: test/App/App.spec.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('My first test suite', function() {
```
Comment (endiny) [inline test/App/App.spec.js:1]: Please, don't forget to add the empty lines to the end of the files

File: package.json [hunk 2]
```diff
@@ -19,6 +21,7 @@
     "@babel/register": "^7.10.5",
     "@babel/runtime": "^7.10.5",
     "babel-loader": "^8.1.0",
+    "cross-env": "^7.0.2",
     "jest": "^26.1.0",
     "react": "^16.13.1",
```
Comment (endiny) [inline package.json:26]: Split the dependencies accordingly

File: test/App/App.spec.js (hunk 1 above)
Comment (bandax) [reply test/App/App.spec.js]: done

File: package.json (hunk 2 above)
Comment (bandax) [reply package.json:45]: done.

**Pull request merged**